api/views.py

```diff
@@ -815,3 +820,103 @@ def create(self, request, *args, **kwargs):
         except Exception as err:
             print(err)
         return HttpResponse("Created new template: {}.".format(name), status=200)
+
+
+@api_view(["GET", "POST"])
+@permission_classes(
+    (
+        IsAuthenticated,
+        AdminPermission,
+    )
+)
+def update_app_status(request):
+    """
+    Manages the app instance status.
+    Implemented as a DRF function based view.
+    Supports GET and POST verbs.
+
+    The service contract for the POST actions is as follows:
+    :param release str: The release id of the app instance.
+    :param new-status str: The new status code.
+    :param event-ts timestamp: A JSON-formatted timestamp, e.g. 2024-01-25T16:02:50.00Z.
+    :param event-msg json dict: An optional json dict containing pod-msg and/or container-msg.
+    :returns: An http status code and status text.
+    """
+
+    # POST verb
+    if request.method == "POST":
+        print("INFO: API method update_app_status called with POST verb.")
+
+        release = None
+        new_status = None
+        event_msg = None
+        event_ts = None
```

Comment:
I think these 4 lines are not doing anything. I there is a KeyError or Exception, then they are not used. If no exception is raised, then it was possible to fetch the values from the request and non of them are `None`. 
```suggestion

```

Reply:
True, except for event_msg. I will remove the others.